hugegraph-api/src/main/java/org/apache/hugegraph/config/ServerOptions.java

```diff
@@ -264,4 +264,35 @@ public static synchronized ServerOptions instance() {
                     disallowEmpty(),
                     true
             );
+
+
+    public static final ConfigOption<String> ARTHAS_TELNET_PORT =
+        new ConfigOption<>(
+            "arthas.telnetPort",
+            "arthas provides telnet ports to the outside",
+            disallowEmpty(),
+            "8562"
+        );
```

Comment:
a blank line is expected here

Reply:
ok, has changed:
![image](https://github.com/apache/incubator-hugegraph/assets/48077841/a56133ee-6f8f-4104-8782-5c6654732183)